### ISSUE TITLE
comparison with CORDEX-CMIP5

### ISSUE THREAD

**Issue by larsbuntemeyer** [euro-cordex/joint-evaluation#18]
@jesusff mentioned that we might compare ERA5 downscaling to CORDEX-CMIP5 ERA-INTERIM. 
@JavierDiezSierra Are you planning to move CORDEX ESGF data to jsc-cordex? Another option would be to upload it to our [S3 bucket on AWS](https://registry.opendata.aws/euro-cordex). I have some sponsorship to host more data, so it would be available globally and it would be easier to reproduce our analysis in the future independently from jsc-cordex.

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer Yes, my plan was to move ERA-INTERIM and CMIP5-CORDEX-EUR-11 to jsc-cordex and include the scripts in the [evaltools](https://github.com/euro-cordex/evaltools) repository. I believe this should not be very demanding in terms of storage requirements, as only the evaluation scenarios at monthly temporal aggregation are needed.

That said, it is fine with me if you think it is preferable to use AWS for storing future scenarios for other analyses, although I am not very familiar with AWS :-)

**Comment by larsbuntemeyer** [comment]
Thank @JavierDiezSierra  for collectin the data in jsc-cordex. I'll have a look into what we actually really need and might be able to also upload it to S3 for the future.

**Comment by JavierDiezSierra** [comment]
@larsbuntemeyer I'm preparing some functions to download CMIP5-CORDEX files using the [esgf-pyclient](https://esgf-pyclient.readthedocs.io/en/latest/quickstart.html) Python package to make everything fully reproducible. I will have it committed during the week and then start the download. :-)

**Comment by larsbuntemeyer** [comment]
Thanks @JavierDiezSierra with the `esgf-pyclient`, you shouldn't need any credentials any if you access CORDEX data on ESGF (that seems to be relaxed).
